tests/suites/hup/suite.js

```diff
@@ -361,6 +361,14 @@ module.exports = {
 			},
 		});
 
+		if(this.suite.options.artifacts !== undefined){
+			// extra artifacts "artifacts" are defined in the config.js of the suite
+			// these artifacts must be in the "suites" directory in the config.js - so you must copy them into suites before
+			// running the tests if these are build time artifacts
+			console.log(`Sending extra artifact folder: ${this.suite.options.artifacts} to worker...`)
+			await this.worker.sendFile(`${__dirname}/${this.suite.options.artifacts}`,'/data/', 'worker');
```

Comment:
```suggestion
			await this.worker.sendFile(`${__dirname}/../suites/${this.suite.options.artifacts}`,'/data/', 'worker');
```

Is this the relative path `${this.suite.options.artifacts}` with respect to the `suites` folder or what? According to that comment, shouldn't the path be this ^^ 

Was writing validation assertions for this in the Leviathan PR and realized this midway.